src/Event/Event.php

```diff
@@ -100,7 +101,7 @@
 use DateTimeImmutable;
 use DateTimeInterface;
 
-class Event extends Offer
+final class Event extends Offer
 {
     protected string $eventId;
```

Comment:
Does this need to be protected?

Reply:
Correct, many properties have `protected` scope but should have `private` scope. I think we need to have a script to fix those things. Maybe something like https://getrector.com/